types/react-transition-group/Transition.d.ts

```diff
@@ -135,7 +135,7 @@ interface BaseTransitionProps<RefElement extends undefined | HTMLElement> {
 }
 
 export type TransitionStatus = typeof ENTERING | typeof ENTERED | typeof EXITING | typeof EXITED | typeof UNMOUNTED;
-export type TransitionChildren = ReactNode | ((status: TransitionStatus) => ReactNode);
+export type TransitionChildren = ReactNode | ((status: TransitionStatus, childProps?: any) => ReactNode);
```

Comment:
[ask] I think `childProps` can be more accurate than using `any` like this:


```suggestion
export type TransitionChildren = ReactNode | ((status: TransitionStatus, childProps?: Record<string, unknown>) => ReactNode);
```

What do you think?

Ref: https://github.com/reactjs/react-transition-group/blob/2989b5b87b4b4d1001f21c8efa503049ffb4fe8d/src/Transition.js#L392

Reply:
👍